README.md

```diff
@@ -25,58 +25,68 @@ _A Git template for bootstrapping a variety of projects with a consistent set of
 1. Update [CONTRIBUTING.md](CONTRIBUTING.md). For more information about open source contributions see [https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors)
 1. Delete any unwanted files in /docker (or the entire /docker folder if you're not containerising)
```

Comment:
Perhaps also a line here about removing the **webproducer** directory if that is not being used in the project?

Reply:
```suggestion
1. Delete any unwanted/unused files or folders such as `/docker`, `/webproducer`, etc.
```